core/src/main/java/com/datastax/oss/driver/internal/core/cql/DefaultPrepareRequest.java

```diff
@@ -127,7 +111,7 @@ public Token getRoutingToken() {
   @Nullable
   @Override
   public Duration getTimeout() {
-    return statement.getTimeout();
+    return null;
```

Comment:
When I originally implemented this, I wasn't sure if it made sense to use the timeout from the `Request`.  I decided to since it currently grabs the timeout from the execution profile, so I figured that made sense here, but i agree that it's somewhat weird because typically prepare requests are fast / don't take any more time based on request.